gcp/opentelemetry-demo-workload-identity.yaml.gotmpl

```diff
@@ -23,3 +23,57 @@ opentelemetry-collector:
     annotations:
       "iam.gke.io/gcp-service-account": "opentelemetry-demo@{{ . }}.iam.gserviceaccount.com"
 {{ end }}
+
+{{ if .Values.collector_image_repository }}
```

Comment:
I believe you need to use `getOrNil` or it will be an error if the variable is unset